src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Security/DoNotAlwaysSkipTokenValidationInDelegates.cs

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Analyzer.Utilities;
+using Analyzer.Utilities.Extensions;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.NetCore.Analyzers.Security
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public sealed class DoNotAlwaysSkipTokenValidationInDelegates : DiagnosticAnalyzer
+    {
+        internal const string DiagnosticId = "CA5405"; // TODO: Somewhere in documentation I saw I need to create a CA5405.md file, don't recall where.
```

Comment:
Found the doc I saw earlier: https://github.com/dotnet/roslyn-analyzers/blob/main/GuidelinesForNewRules.md

Will follow these rules for adding documentation for the two new rules.

Reply:
I believe it's here: [https://github.com/dotnet/docs/tree/main/docs/fundamentals/code-analysis/quality-rules](https://github.com/dotnet/docs/tree/main/docs/fundamentals/code-analysis/quality-rules)